recipes/sqlite3/all/conanfile.py

```diff
@@ -38,6 +39,7 @@ class ConanSqlite3(ConanFile):
         "disable_gethostuuid": [True, False],
         "max_blob_size": "ANY",
         "build_executable": [True, False],
+        "target_os": ["auto", "windows", "unix", "other"],
```

Comment:
It shouldn't be an option, but come from settings.

Reply:
Thanks, you are right. I changed that option to boolean (using default or custom SQLite VFS)